PERF-XDP-TCP-LATENCY: New testcase to verify XDP Latency performance

abhimarathe opened the pull request:
This testcase will use lagscope performance script. I have made changes in perf_lagscope.sh to adopt test IP as in the previous scenarios it was assumed that lagscope will be running on primary IP address.

```[LISAv2 Test Results Summary]
Test Run On           : 07/08/2020 03:57:52
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Initial Kernel Version: 5.7.0
Final Kernel Version  : 5.7.0
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:17

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              PERF-XDP-TCP-LATENCY                                                              PASS                15.78


Logs can be found at C:\LISAv2\UQ54\TestResults\2020-07-07-20-57-45-8715```

abhimarathe committed:
Testscripts/Linux/perf_lagscope.sh
PERF-XDP-TCP-LATENCY: New testcase to verify XDP Latency performance

juhlee-microsoft reviewed (commented):
Can you check out my questions/comment?

LiliDeng reviewed (dismissed):
make sure test pass against your latest code

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

juhlee-microsoft reviewed (approved):
Please fix my comments before merging

abhimarathe committed:
PERF-XDP-TCP-LATENCY.ps1: Fixed comments and removed root dependency

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

abhimarathe committed:
perf_lagscope: Defined "pingIterations" in shell script and removed its use
from ps and xml files.
This has not been used that much but the cost of providing this
compatibility would increase maintenance.

abhimarathe committed:
Is_XDPCompatible: Removed function and added kernel version independent
check using xdp queues.

LiliDeng reviewed (dismissed):
there is one pingIteration in case NESTED-KVM-LAGSCOPE-DIFFERENT-L1-PUBLIC-BRIDGE, maybe remove it?

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

abhimarathe committed:
PERF-XDP-TCP-LATENCY: Removed kernel version check

abhimarathe committed:
Merge branch 'master' into abhimarathe-XDP-PERF

abhimarathe committed:
PERF-XDP-TCP-LATENCY: Updated return value of Compare-Results

abhimarathe committed:
Merge branch 'abhimarathe-XDP-PERF' of https://github.com/LIS/LISAv2 into abhimarathe-XDP-PERF

abhimarathe committed:
PERF-XDP-TCP-LATENCY.ps1: reformated
